scripts/tests/test_validate_unwanted_patterns.py

```diff
@@ -0,0 +1,77 @@
+import io
+
+# the "as" is just so I won't have to change this in many places later.
+import validate_string_concatenation as validate_unwanted_patterns
+
+
+class GoodBarePytestRaises:
```

Comment:
You don't need this class, or the Bad one, that's something that makes sense for the docstings (we have a class with methods, so we can extract the docstrings from them). But not here, just define the values in the tests when you need them.